ui/components/app/account-menu/keyring-label.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { KEYRING_TYPES } from '../../../../shared/constants/hardware-wallets';
+
+export default function KeyRingLabel(props) {
+  const t = useI18nContext();
+
+  const { keyring } = props;
+
+  let label = null;
+
+  // Keyring value might take a while to get a value
+  if (!keyring) {
+    return null;
+  }
+
+  const { type } = keyring;
+
+  switch (type) {
+    case KEYRING_TYPES.QR:
+      label = 'QR';
```

Comment:
Maybe in `shared/constants/hardware-wallets.js` we should create a `KEYRING_NAMES` object that has these labels; I feel like we'll end up using these shorthand names more.  I use Ledger, for example, in an upcoming PR.